components/cellery-component-test/pom.xml

```diff
@@ -0,0 +1,266 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>io.cellery</groupId>
+        <version>0.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+        <artifactId>cellery-component-parent</artifactId>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>cellery-component-test</artifactId>
+    <packaging>jar</packaging>
+    <name>Cellery - Cellery Component Test</name>
+    <url>https://wso2-cellery.github.io</url>
+
+    <dependencies>
```

Comment:
Please verify whether you need all these dependencies. It looks like some of them are not required for this module.